### PR TITLE
LocalBearTestHelper.py: Docstring Indentation

### PR DESCRIPTION
**Pull request by pushkalkatara**
**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!
Solves issue #5146

### CODE REVIEW

File: coalib/testing/LocalBearTestHelper.py [hunk 1]
```diff
@@ -123,14 +123,21 @@ def check_validity(self,
...
+            The local bear to check with.
+        :param lines:
+            The lines to check. (List of strings)
+        :param filename:
+            The filename, if it matters.
+        :param valid:
+            Whether the lines are valid or not.
+        :param force_linebreaks:
+            Whether to append newlines at each line if needed.
+            (Bears expect a \\n for every line)
```
Comment (newbazz) [inline coalib/testing/LocalBearTestHelper.py]: make it inline, i dont think it will exceed the limit.

Comment (pushkalkatara) [reply coalib/testing/LocalBearTestHelper.py]: It was exceeding 2 characters.

**Comment by newbazz** [comment]
Why closed the PR?

**Comment by pushkalkatara** [comment]
Sorry @newbazz Next time i will not open new pull requests, i just learnt git commit --amend.

**Pull request closed without merging**